app/src/androidTest/java/org/simple/clinic/DatabaseMigrationAndroidTest.kt

```diff
@@ -352,15 +359,11 @@ class DatabaseMigrationAndroidTest {
     // We need to do this here because Room does not create virtual tables for us.
     db_v15.execSQL("""CREATE VIRTUAL TABLE "PatientFuzzySearch" USING spellfix1""")
 
-    db_v15.query("""SELECT DISTINCT "tbl_name" FROM "sqlite_master" WHERE "tbl_name"='PatientFuzzySearch'""").use {
-      assertThat(it.count).isEqualTo(1)
-    }
+    db_v15.assertTableExists("PatientFuzzySearch")
```

Comment:
Should this be a part of this PR/commit?

Reply:
We could perhaps break this commit into two?

Reply:
Done.